### DISCUSSION TITLE
Helping you upgrade

### DISCUSSION THREAD

**Discussion by lg-nineoneninethreenine** [kylereddoch/kylereddoch.github.io#3] [General]
I saw your comment and hopped on here. Seems like you changed remote_theme to theme. Try changing it back and if it fails, I'll try a PR on Saturday.

**Comment by lg-nineoneninethreenine** [comment]
You could cross-refer with my setup if you like.

https://im-coder-lg.github.io

**Reply by kylereddoch** [reply]
Yeah I have it as remote theme locally, just haven't pushed to to GitHub. Although, before, I did have remote theme set and was getting build errors. When I get home I'll send screenshots of the errors.
